src/custom-element-widget.js

```diff
@@ -170,5 +208,6 @@ module.exports = {
 
   makeConstructor,
   makeInit,
-  makeUpdate
+  makeUpdate,
+  makeDestroy
```

Comment:
'object short notation' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).